ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -176,11 +184,11 @@ function RequestCVEsTable({
 
                                 const cveURLQueryOptions = {
                                     s: {
-                                        IMAGE: searchFilter.Image,
+                                        IMAGE: queryObject.Image,
                                     },
                                 };
 
-                                const cveURL = `${vulnMgmtBaseUrl}/${getWorkloadEntityPagePath(
+                                const cveURL = `${vulnerabilitiesWorkloadCvesPath}/${getWorkloadEntityPagePath(
```

Comment:
> With the addition of the "All images" view, the tabs are no longer necessary and complicate the UI.

Does this need feature flag?
* address with deprecated path if disabled
* address with **All vulnerabile images** path if enabled

Reply:
Not strictly, when the feature flag is enabled the redirect will be put in place that automatically routes this to "All vulnerable images".

I do want to make this explicit in a follow up, but didn't prioritize it highly since:
1. Functionality it will be equivalent
2. We will likely need to maintain the redirect for some time regardless of this change